.buildkite/longruns/pipeline.yml

```diff
@@ -34,7 +34,7 @@ steps:
       # TODO: This can be removed once all the branches that use MPIPreferences
       # are deleted
       - echo "--- Remove MPIPreferences"
-      - "rm -f ${JULIA_DEPOT_PATH}/environments/v1.9/LocalPreferences.toml"
+      - "rm -f ${JULIA_DEPOT_PATH}/environments/v1.10/LocalPreferences.toml"
```

Comment:
You can remove this section.

The `rm` was there while there were unupdated branches, but nothing will write the wrong entries to `v1.10/LocalPreferences.toml`.